packages/bundler/src/test/html-bundler_test.ts

```diff
@@ -33,6 +31,32 @@ const stripSpace = (html: string): string =>
 
 suite('HtmlBundler', () => {
 
+  test('inline es6 modules with node module-resolution', async () => {
+    const root = 'test/html/inline-es6-modules';
+    const analyzer = new Analyzer({
+      urlResolver: new FsUrlResolver(root),
+      urlLoader: new FsUrlLoader(root),
+      moduleResolution: 'node'
+    });
+    const bundler = new Bundler({analyzer});
+    const usesNpmPackageUrl = analyzer.resolveUrl('uses-npm-package.html')!;
```

Comment:
For clarity, I think it'd be better to talk about using node module resolution rather than using npm packages, in these variable and file names. Node module resolution is the terminology used in the rest of the system for this

Reply:
Renamed to `import-using-node-module-resolution.html`